main.go

```diff
@@ -137,10 +137,11 @@ func mainWithError() (err error) {
 
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
-	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.Address, "", "Address used to connect to Kubernetes. When empty in-cluster config is created.")
 	daemonCommand.PersistentFlags().String(f.Service.CNR.Address, "https://quay.io", "Address used to connect to CNR, defaults to quay's managed offering.")
 	daemonCommand.PersistentFlags().String(f.Service.CNR.Organization, "giantswarm", "CNR organization.")
-	daemonCommand.PersistentFlags().Bool(f.Service.Kubernetes.InCluster, true, "Whether to use the in-cluster config to authenticate with Kubernetes.")
+	daemonCommand.PersistentFlags().String(f.Service.Helm.Host, ":44134", "Host and port for Helm client to connect to the Tiller endpoint.")
```

Comment:
Would be good to have an integration test for checking the connection, in the current tests tiller is already set up available.

Reply:
Definitely! I'd like to tackle that after I've got GetCurrentState working. I'd like to get my feet wet with integration testing :)